src/utils/global/ambassadorCard/ambassadorCard.module.scss

```diff
@@ -1,16 +1,18 @@
 @import '../../../variables.scss';
 
 .ambassadorCard {
-  width: 295px;
-  height: 460px;
-  max-height: 100vh;
+  width: 18.5rem;
+  max-width: 100%;
+  min-height: min(28.5rem, 100%);
+  max-height: 100%;
```

Comment:
`max-height` percents are based on the scaled height so it ends up less high than it needs to in small viewports. I came up with this workaround.  There probably is a better way though:

```scss
  @media screen and (max-height: 555px) {
    max-height: calc(100% / 0.9);
  }
  @media screen and (max-height: 500px) {
    max-height: calc(100% / 0.8);
  }
  @media screen and (max-height: 450px) {
    max-height: calc(100% / 0.7);
  }
  @media screen and (max-height: 410px) {
    max-height: calc(100% / 0.6);
  }
```

Reply:
As discussed in Discord, not going to fix this in this PR, but will open an issue/PR to revisit scaling holistically, with the ideal being to remove the use of scale and instead change the font-size at breakpoints, with the whole extension using rems so everything scales accordingly.